burst/src/main/java/com/squareup/burst/TestConstructor.java

```diff
@@ -113,4 +114,17 @@ public void initializeFieldsOnInstance(final Object instance, final Object[] arg
       fields[i].set(instance, args[i]);
     }
   }
+
+  public Annotation[][] getArgumentAnnotations() {
+    final Annotation[][] ctorAnnotations = constructor.getParameterAnnotations();
```

Comment:
I'm not sure I follow what you mean here. parameter count isn't necessarily indicative of annotations count